app/lib/features/news/widgets/news_post_editor/select_action_item.dart

```diff
@@ -23,6 +25,13 @@ class SelectActionItem extends StatelessWidget {
           onTap: () => onShareEventSelected(),
         ),
         const SizedBox(height: 20),
+        actionItemUI(
+          context: context,
+          actionIcon: Atlas.pin,
+          actionName: L10n.of(context).sharePin,
+          onTap: () => onSharePinSelected(),
```

Comment:
```suggestion
          onTap: onSharePinSelected,
```

no need to wrap again